src/OctoshiftCLI.Tests/Octoshift/Services/GithubApiTests.cs

```diff
@@ -1436,7 +1439,56 @@ public async Task GetMigrationLogUrl_Returns_The_Migration_Log_URL()
         var expectedMigrationLog = await _githubApi.GetMigrationLogUrl(GITHUB_ORG, GITHUB_REPO);
 
         // Assert
-        expectedMigrationLog.Should().Be(migrationLogUrl);
+        expectedMigrationLog.Should().Be((migrationLogUrl, migrationId));
+    }
+
+    [Fact]
+    public async Task GetMigrationLogUrl_Returns_Empty_String_Migration_Log_URL_If_Returned_From_The_API()
```

Comment:
This test seems unnecessary. It's just testing that it returning the empty log url value. Since an empty value is still an acceptable value it has already been covered via GetMigrationLogUrl_Returns_The_Migration_Log_URL_And_Migration_ID`